linkerd/protocol/h2/src/main/scala/io/buoyant/linkerd/protocol/h2/H2AccessLogger.scala

```diff
@@ -35,6 +35,11 @@ case class H2AccessLogger(log: Logger) extends SimpleFilter[Request, Response] {
 object H2AccessLogger {
 
   object param {
+    case class Label(label: String)
```

Comment:
I don't think we need a new label param, we can just use the existing one (com.twitter.finagle.param.Label)